.ci/StarterRuleSet-AllRulesByCategory.groovy.txt

```diff
@@ -347,7 +348,7 @@ ruleset {
     ObjectFinalize
     PublicFinalizeMethod
     SystemExit {
-        doNotApplyToFileNames = 'pitest-survival-check-xml.groovy'
+        doNotApplyToFileNames = '*.groovy'
```

Comment:
Can we enumerate files here instead of wildcard?

Reply:
Done